Jenkinsfile

```diff
@@ -2,6 +2,13 @@
 
 @Library('xmos_jenkins_shared_library@v0.39.0') _
 
+def clone_test_deps() {
+  dir("${WORKSPACE}") {
```

Comment:
best to add this to requirements, rather than Jenkinfile

Make it easier to run locally.